modules/lightmapper_rd/lm_compute.glsl

```diff
@@ -118,13 +118,17 @@ bool ray_hits_triangle(vec3 from, vec3 dir, float max_dist, vec3 p0, vec3 p1, ve
 bool trace_ray(vec3 p_from, vec3 p_to
 #if defined(MODE_BOUNCE_LIGHT) || defined(MODE_LIGHT_PROBES)
 		,
-		out uint r_triangle, out vec3 r_barycentric
+		out uint r_triangle, out vec3 r_barycentric, out bool backface
```

Comment:
By convention, parameters which are modified by the function to return values use the `r_` prefix (like the two other `out` parameters here).

Reply:
Thanks for the feedback, I updated the PR and removed the extra parameter.